tests/model_hub_tests/performance_tests/conftest.py

```diff
@@ -26,27 +27,32 @@ def pytest_runtest_makereport(item, call):
 @pytest.mark.optionalhook
 def pytest_html_results_table_header(cells):
     cells.insert(2, html.th('status', class_="sortable"))
-    cells.insert(3, html.th('converted model infer time'))
+    cells.insert(3, html.th('converted model infer time secs'))
     cells.insert(4, html.th('converted model infer time variance'))
-    cells.insert(5, html.th('read model infer time'))
-    cells.insert(6, html.th('read model infer time variance'))
-    cells.insert(7, html.th('model infer time ratio converted_model_time/read_model_time'))
-
-
-def round_num(n: float) -> str:
-    s = '{:.4E}'.format(n)
-    if s.endswith('E+00'):
-        return s[:-4]
-    return s
+    cells.insert(5, html.th('converted model heat n repeats'))
+    cells.insert(6, html.th('converted model measurement n repeats'))
+    cells.insert(7, html.th('converted model throughput'))
+    cells.insert(8, html.th('read model infer time secs'))
```

Comment:
I believe we should leave only "infer time", "variance" and maybe "measurement repeats" in the html report